src/vstest.console/Internal/MatcherHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Internal
+{
+    using Microsoft.Extensions.FileSystemGlobbing;
+    using Microsoft.Extensions.FileSystemGlobbing.Abstractions;
+    using vstest.console.Internal.Interfaces;
+
+    /// <summary>
+    /// Class for implementing the FileSystemGlobbing Matcher class methods
+    /// </summary>
+    internal class MatcherHelper: IMatcherHelper
+    {
+        private Matcher matcher;
+
+        public MatcherHelper()
+        {
+            this.matcher = new Matcher();
```

Comment:
>Matcher [](start = 31, length = 7)

see if Matcher can be mocked.

Reply:
It can be mocked. Removed interface and helper classes